apps/i18n/studio/en_us.json

```diff
@@ -2,6 +2,12 @@
   "actor": "actor",
   "addCharacter": "add a",
   "addCharacterTooltip": "Add a character to the scene.",
+  "addPoints10": "add 10 points",
```

Comment:
blockly determines which part of the block should be a combo box - this approach is very flexible for localization. See `moveDistance25` etc. for another example